sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -217,7 +220,7 @@ public Read withBigtableOptions(BigtableOptions.Builder optionsBuilder) {
       BigtableOptions optionsWithAgent =
           clonedBuilder.setUserAgent(getBeamSdkPartOfUserAgent()).build();
 
-      return new Read(optionsWithAgent, tableId, keyRange, filter, bigtableService);
+      return new Read(optionsWithAgent, tableId, keyRange, filter, null, null,  bigtableService);
```

Comment:
Not sure how to fix it... but all these nulls make seem very lilkely to be error prone. I believe that to be consistent with the PTransform style guide[1] we'd be better off having a single "ReadBuilder" that had a bunch of default,s and then having the various options fill in using fluent builder methods.

[1] https://beam.apache.org/contribute/ptransform-style-guide/#fluent-builders-for-configuration